src/pages/User/Learning/Points/Points.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
```

Comment:
już tutaj to zostawmy, ale na przyszłość lepiej do PR z całkiem nową funkcjonalnośćią jakąs jest zustand lepiej nie dodawać jakiegoś nowego komponentu. Później nie będziesz wiedziała co powoduje potencjalne błędy gdyby coś się posypało.  Najlepiej zrobić nowy branch, nowy PR i osobno to mergować